Support/CMake/Toolchain-WinStore-ARM-Toolset-x86_64.cmake

```diff
@@ -0,0 +1,28 @@
+##
```

Comment:
This file is completely unnecessary, let's ignore the toolset BS and just let cmake pick the default.